src/graphql/models/Article.js

```diff
@@ -529,6 +530,21 @@ const Article = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Attachment hash to search or identify files',
     },
+    cooccurrences: {
+      type: new GraphQLList(new GraphQLNonNull(Cooccurrence)),
+      resolve: async ({ id }, args, { loaders }) =>
+        loaders.searchResultLoader.load({
+          index: 'cooccurrences',
+          type: 'doc',
+          body: {
+            query: {
+              bool: {
+                should: [{ term: { articleIds: id } }],
+              },
+            },
```

Comment:
Since there is only 1 element in bool query, we can use term query directly

```suggestion
            query: {
              term: { articleIds: id },
            },
```

Reply:
updated